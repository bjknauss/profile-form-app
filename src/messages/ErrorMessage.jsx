import React from 'react'
import withMessage from './MessageWrapper'

let ErrorMessage = ({message}) => (
  <div className="alert alert-danger">
    {message}
  </div>
)

ErrorMessage = withMessage(ErrorMessage)

export default ErrorMessage