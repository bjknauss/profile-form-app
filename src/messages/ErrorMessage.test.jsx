import React from 'react'
import renderer from 'react-test-renderer'
import ErrorMessage from './ErrorMessage'

test('Displays with error message', () => {
  const errorMessage = "test error msg"
  const component = renderer.create(
    <ErrorMessage message={errorMessage} />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})