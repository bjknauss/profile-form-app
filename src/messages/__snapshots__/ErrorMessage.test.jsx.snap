// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays with error message 1`] = `
<div
  className="alert alert-danger"
>
  test error msg
</div>
`;
